### PR TITLE
Feature/publishing

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Last auto publish PR

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -65,3 +65,94 @@ jobs:
                       cd progress
                       yarn test:demos
                       cd ..
+            - persist_to_workspace:
+                  root: .
+                  paths:
+                    - iconfont
+                    - design
```
Comment (joebochill) [inline .circleci/config.yml:72]: I don't think we need to persist 'design' folder unless we are doing the build step again later

File: progress/angular/components/PUBLISHING.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Publishing Instructions
+
+To update the version number, edit the version in `package.json`. 
+
+To publish a new package through NPM, run the following commands from the root folder: 
+
+```
+yarn build
+cd ../dist
```
Comment (joebochill) [inline progress/angular/components/PUBLISHING.md:9]: is the dist folder a directory above where you run the build command?

**Review by joebochill — APPROVED** [review]
Looks good. Did you try testing this out at all in your personal npm?

File: .circleci/config.yml (hunk 1 above)
Comment (emclaug2) [reply .circleci/config.yml:72]: If the design folder isn't something that is generated you're correct, I'll remove it from the persist list.

File: progress/angular/components/PUBLISHING.md (hunk 2 above)
Comment (emclaug2) [reply progress/angular/components/PUBLISHING.md:9]: Yep, the dist folder is generated 1 folder above where the build command is ran.  I'm not sure why, but I was just looking at how things currently work.

**Review by joebochill — APPROVED** [review]

**Pull request merged**